src/main/native/CCM.c

```diff
@@ -6,7 +6,7 @@
  * in the file LICENSE in the source distribution.
  */
 
-typedef int (*PFI)();
+typedef int (*PFI)(void);
```

Comment:
This was used for the definition of the functions later on. Do we still need it?

Reply:
We still seem to need these as PFI is used in the defintion of FUNC which is then used through the code. I did move this PFI defintion to a new header file named `zHardwareFunctions.h` so its now only defined in one spot although i did not try to eliminate it.